pkg/setting/setting_sentry.go

```diff
@@ -11,12 +11,15 @@ type Sentry struct {
 
 func (cfg *Cfg) readSentryConfig() {
 	raw := cfg.Raw.Section("log.frontend")
-	cfg.Sentry = Sentry{
-		Enabled:        raw.Key("enabled").MustBool(true),
-		DSN:            raw.Key("sentry_dsn").String(),
-		CustomEndpoint: raw.Key("custom_endpoint").String(),
-		SampleRate:     raw.Key("sample_rate").MustFloat64(),
-		EndpointRPS:    raw.Key("log_endpoint_requests_per_second_limit").MustInt(),
-		EndpointBurst:  raw.Key("log_endpoint_burst_limit").MustInt(),
+	provider := raw.Key("provider").MustString("sentry")
+	if provider == "sentry" || provider != "grafana" {
+		cfg.Sentry = Sentry{
+			Enabled:        raw.Key("enabled").MustBool(true),
+			DSN:            raw.Key("sentry_dsn").String(),
+			CustomEndpoint: raw.Key("custom_endpoint").MustString("/log"),
+			SampleRate:     raw.Key("sample_rate").MustFloat64(),
+			EndpointRPS:    raw.Key("log_endpoint_requests_per_second_limit").MustInt(),
+			EndpointBurst:  raw.Key("log_endpoint_burst_limit").MustInt(),
+		}
```

Comment:
Same question as above: wouldn't it be better to fallback to the default values?

Reply:
Very good catch, added https://github.com/grafana/grafana/pull/50801/commits/514c9dff0baa711644990441a78c9f99d2ba4396